src/App.tsx

```diff
@@ -1,61 +1,138 @@
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
```

Comment:
Consider removing commented out code. It can be confusing and make the code harder to read. 🤔